package/upgrade/upgrade_manifests.sh

```diff
@@ -1161,6 +1239,7 @@ detect_upgrade
 check_version
 pre_upgrade_manifest
 pause_all_charts
+skip_restart_rancher_system_agent
```

Comment:
let's add proper comments and issue link for reference, also prefer to check only run it if the previous Harvester version is less than `v1.2.0`.

Reply:
I have added the version check, PTAL. Thank you!